SingularityBase/src/main/java/com/hubspot/singularity/SingularityEmailType.java

```diff
@@ -1,6 +1,18 @@
 package com.hubspot.singularity;
 
 public enum SingularityEmailType {
-  TASK_LOST, TASK_KILLED, TASK_FINISHED_SCHEDULED, TASK_FINISHED_LONG_RUNNING, TASK_FINISHED_ON_DEMAND, TASK_FINISHED_RUN_ONCE, TASK_FAILED, TASK_SCHEDULED_OVERDUE_TO_FINISH,
-  TASK_KILLED_DECOMISSIONED, TASK_KILLED_UNHEALTHY, REQUEST_IN_COOLDOWN, SINGULARITY_ABORTING, REQUEST_REMOVED, REQUEST_PAUSED, REQUEST_UNPAUSED, REQUEST_SCALED, TASK_FAILED_DECOMISSIONED
+  TASK_LOST("#D9534F"), TASK_KILLED("#777777"), TASK_FINISHED_SCHEDULED("#5CB85C"), TASK_FINISHED_LONG_RUNNING("#5CB85C"),
```

Comment:
If at any point in the future it is desired to change the color of one of the email types, this is the place to change it.